pwa/src/templates/organizationsTemplate/OrganizationsTemplate.module.css

```diff
@@ -0,0 +1,121 @@
+.container > *:not(:last-child) {
+  margin-block-end: var(--web-app-size-xl);
+}
+
+.header > :not(:last-child) {
+  margin-block-end: var(--web-app-size-lg);
+}
+
+.filtersAndResultsContainer {
+  display: flex;
+  flex-direction: column;
+  gap: var(--web-app-size-lg);
+}
+
+.results {
+  flex: 3;
+
+  --utrecht-heading-4-font-weight: normal;
+  --utrecht-heading-4-font-size: var(--web-app-font-size-md);
+  --utrecht-heading-4-distanced-margin-block-end: var(--web-app-size-2xs);
+  --utrecht-heading-4-distanced-margin-block-start: var(--web-app-size-2xs);
```

Comment:
I think this is a copy paste from the components template, but some of this logic does not apply here.

E.g.: there's no two columns, so setting the flex: 3 1, and flex-direction is redundant here.

I'd suggest simply setting the width to 100%.